chrome/content/zotero/zoteroPane.xul

```diff
@@ -106,7 +106,19 @@
 					<toolbarbutton id="zotero-tb-library-add-menu" class="zotero-tb-button" tooltiptext="&zotero.toolbar.newLibrary.label;" type="menu">
 						<menupopup id="zotero-tb-library-add-popup">
 							<menuitem id="zotero-tb-group-add" label="&zotero.toolbar.newGroup;" oncommand="ZoteroPane_Local.newGroup()"/>
-							<menuitem id="zotero-tb-feed-add" label="&zotero.toolbar.newFeed.label;" command="cmd_zotero_newFeed"/>
+							<menu id="zotero-tb-feed-add-menu" label="&zotero.toolbar.feeds.new;">
```

Comment:
We want both the menuitem and the menu here. In Standalone, there should only be the "New Feed…" menuitem, with ("From Page…") implied since that's the only option. (That's what I meant about having the id on the menu, so it can be used for hiding in Standalone.)